docs/production/upgrade-or-modify.md

```diff
@@ -251,14 +251,76 @@ and the latter for `server` contexts.
 ## Upgrading the operating system
 
 When you upgrade the operating system on which Zulip is installed
-(E.g. Ubuntu 18.04 Bionic to Ubuntu 20.04 Focal), you need to take
+(E.g. Ubuntu 20.04 Focal to Ubuntu 22.04 Jammy), you need to take
 some additional steps to update your Zulip installation, documented
 below.
 
 The steps are largely the same for the various OS upgrades aside from
 the versions of PostgreSQL, so you should be able to adapt these
 instructions for other supported platforms.
 
+### Upgrading from Ubuntu 20.04 Focal to 22.04 Jammy
+
+1. Upgrade your server to the latest Zulip `3.x` or `4.x` release (at
+   least 3.0, which adds support for Ubuntu 20.04 and above). You can only
+   upgrade to Zulip 5.0 and newer after completing this process, since
+   newer releases don't support Ubuntu 18.04.
```

Comment:
The first sentence should be updated: “Upgrade your server to the latest Zulip `5.x` release (at least 5.3, which adds support for Ubuntu 22.04 and above).” The second sentence should be deleted, as the current release still supports Ubuntu 20.04.

Reply:
Perfect, thanks!